fury/effects/effects.py

```diff
@@ -187,14 +188,50 @@ def colormap_to_texture(
 
 
 class KDE():
+    """
+    Class that implements the Kernel Density Estimation effect of a given set of points.
+
+    Parameters
+    ----------
+    points : np.ndarray (N, 3)
+        Array of points to be displayed.
+    bandwidths : np.ndarray (1, ) or (N, 1)
+        Array of bandwidths to be used in the KDE calculations. Must be one or one for each point.
+    kernel : str, optional
+        Kernel to be used for the distribution calculation. The available options are:
+        * "cosine"
+        * "epanechnikov"
+        * "exponential"
+        * "gaussian"
+        * "linear"
+        * "tophat"
+    opacity : float, optional
+        Opacity of the effect, defined between [0.0, 1.0].
+    colormap : str, optional.
+        Colormap matplotlib name for the KDE rendering. Default is "viridis".
+    custom_colormap : np.ndarray (N, 4), optional
+        Custom colormap for the KDE rendering. Default is none which means no
+        custom colormap is desired. If passed, will overwrite matplotlib colormap
+        chosen in the previous parameter.
+    """
+
     def __init__(self,
-            points : np.ndarray,
-            bandwidths,
-            kernel : str = "gaussian",
-            opacity : float = 1.0,
-            colormap : str = "viridis",
-            custom_colormap : np.array = None):
-        self.points = points
+                 points : np.ndarray,
+                 bandwidths,
```

Comment:
Why there is no type specified for `bandwidth`?

Reply:
It is because also allow the passing of single values, not only `np.ndarrays`. I guess I just need to change the description of it to be more clear, what do you think?

Reply:
@filipinascimento for some reason `Union` is outputting the following error:

  File "...", line 220, in KDE
    bandwidths : Union(float, np.ndarray),
  File "...", line 387, in __call__
    raise TypeError(f"Cannot instantiate {self!r}")
TypeError: Cannot instantiate typing.Union

Do you know how to fix it?

Reply:
Ops it is tUnion[float, np.ndarray]